geo/map_processing/graph.py

```diff
@@ -13,23 +13,18 @@
 # limitations under the License.
 
 from s2geometry import pywraps2 as s2
+from s2geometry.pywraps2 import S2Cell
```

Comment:
Remove this import and use s2.S2Cell in code.

Reply:
done